docs/install.rst

```diff
@@ -14,153 +14,148 @@ Once installation is complete, you can run :ref:`mitmproxy`, :ref:`mitmdump` or
 Installation on macOS
 ---------------------
 
-You can use Homebrew to install everything:
+The recommended way to install mitmproxy on macOS is to use `Homebrew`_:
 
 .. code:: bash
 
     brew install mitmproxy
 
-Or you can download the pre-built binary packages from our `releases`_.
+Alternatively you can download our self-contained pre-built binary packages from
+our `releases`_ page.
 
 
 .. _install-windows:
 
 Installation on Windows
 -----------------------
 
-The recommended way to install mitmproxy on Windows is to use the installer
-provided at `mitmproxy.org`_. After installation, you'll find shortcuts for
-:ref:`mitmweb` (the web-based interface) and :ref:`mitmdump` in the start menu.
-Both executables are  added to your PATH and can be invoked from the command
-line.
+The recommended way to install mitmproxy on Windows is to download our
+self-contained pre-built binary packages from our `releases`_ page.
+
+After installation, you'll find shortcuts for :ref:`mitmweb` (the web-based
+interface) and :ref:`mitmdump` in the start menu. Both executables are added to
+your PATH and can be invoked from the command line.
 
 .. note::
-    Mitmproxy's console interface is not supported on Windows, but you can use
-    mitmweb (the web-based interface) and mitmdump.
+    The console interface `mitmproxy` is not supported on Windows, but you can
+    use mitmweb (the web-based interface) and mitmdump.
```

Comment:
Maybe `Mitmproxy's console interface (``mitmproxy``) is not supported...`?

Reply:
My main intention here was, the fact that I don't like `Mitmproxy` spelled with a capital `M`.